RHEL/7/input/xccdf/system/selinux.xml

```diff
@@ -5405,4 +5405,22 @@ If this setting is enabled, it should be disabled.
 
 </Group>
 
+<Rule id="docker_selinux_enabled" severity="high">
+<title>Ensure SELinux support is enabled in Docker</title>
+<description>
+To enable the SELinux for Docker, the Docker service must be configured
+to run the Docker daemon with <tt>--selinux-enabled</tt> option.
+In <tt>/etc/sysconfig/docker</tt>  add or correct the following line to enable SELinux support in
+Docker daemon.
```

Comment:
This should say the following:

To enable SELinux for the Docker service, the Docker service must be configured
to run the Docker daemon with the <tt>--selinux-enabled</tt> option.
In the <tt>/etc/sysconfig/docker</tt> configuration file, add or correct the following line to enable SELinux support in the Docker daemon: